README.pg_sphere

```diff
@@ -13,7 +13,21 @@ This is an R-Tree implementation using GiST for spherical objects like
 spherical points and spherical circles with useful functions and operators.
 
 NOTICE:
-     This version will work only with PostgreSQL version 9.6 and above.
+     This version will work only with PostgreSQL version 10 and above.
+
+VERSIONING:
+
+Stable versions are marked with tags containing version numbers in the GitHub
+repository at https://github.com/postgrespro/pgsphere/. Each stable version
+contains upgrade scripts for updating an existing installation to a newest
```

Comment:
Oops, "a newest" should be "the newest" on this line. Other than that, looks good!

Reply:
@esabol may be "the latest"?

Reply:
Yes, that works too. It's better.